### PR TITLE
EW-492: Add update person endpoint

### PR DESCRIPTION
**Pull request by mkreuzkam-cap**
# Description
- Add update person endpoint

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/EW-492

## Approval for review
- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### CODE REVIEW

File: src/modules/person/api/update-person.body.params.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    @IsString()
+    @IsEnum(Geschlecht)
+    @ApiProperty({ enum: Geschlecht, required: false })
+    public readonly geschlecht?: Geschlecht;
+
+    @AutoMap()
+    @IsOptional()
+    @IsString()
+    @ApiProperty({ default: 'de-DE', required: false })
+    public readonly lokalisierung?: string = 'de-DE';
```
Comment (alweber-cap) [inline src/modules/person/api/update-person.body.params.ts:46]: is this default in the SchulConneX documentation?

**Review by alweber-cap — APPROVED** [review]

File: src/modules/person/api/update-person.body.params.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    @AutoMap()
+    @IsOptional()
+    @IsBoolean()
+    @ApiProperty({ required: false })
+    public readonly auskunftssperre?: boolean;
+
+    @AutoMap()
+    @IsString()
+    @ApiProperty()
+    public readonly revision!: string;
```
Comment (Fshmit) [inline src/modules/person/api/update-person.body.params.ts:64]: revision should always be set in the request or not? 
@ApiProperty({ required: true})

Comment (alweber-cap) [reply src/modules/person/api/update-person.body.params.ts:64]: according to spec yes. good catch!

Comment (mkreuzkam-cap) [reply src/modules/person/api/update-person.body.params.ts:64]: I added {required: true}

**Review by Fshmit — APPROVED** [review]

**Pull request merged**